dev/camdenhells/REQUIREMENTS.md

```diff
@@ -328,7 +329,7 @@ This does not preclude performance impacts; clearly there will be limits, but ha
   * Isolated - from all (other) render thread/event processing
     * Frame draws, window re-sizes, queued events, other CRUD transactions, etc.
     * Implies running on the render thread as an event, queued from any thread
-    * TODO how to handle queued transactions after window detach/loss of context...
+    * Queued transactions after window detach/loss of context are dropped
```

Comment:
Right, so then it is purely the client's responsibility to handle this situation if/as needed.

Reply:
Yes